apis/dispatch.php

```diff
@@ -414,9 +414,9 @@
 try {
     (new TelemetryService())->trackApiRequestEvent([
         'eventType' => 'API',
-        'eventLabel' => json_encode($GLOBALS['oauth_scopes'] ?? []),
-        'eventUrl' => $resource,
-        'eventTarget' => $restRequest->getRequestMethod() . ' ' . $_SESSION['api'],
+        'eventLabel' => strtoupper($_SESSION['api'] ?? 'UNKNOWN'),
+        'eventUrl' => $restRequest->getRequestMethod() . ' ' . $resource,
+        'eventTarget' => json_encode($GLOBALS['oauth_scopes'] ?? []),
```

Comment:
@sjpadgett , just put in your code for this fix. Also made the parameter type an array instead of mixed. and dropped reportUsageData case from track_events.php since realized not being used anywhere.